topology/probes/k8s/linker.go

```diff
@@ -76,7 +82,7 @@ func (l *ABLinker) GetBALinks(bNode *graph.Node) (edges []*graph.Edge) {
 }
 
 // NewABLinker create and initialize an ABLinker based linker
-func NewABLinker(g *graph.Graph, aManager, aType, bManager, bType string, areLinked AreLinked) probe.Probe {
+func NewABLinker(g *graph.Graph, aManager, aType, bManager, bType string, getMetadata GetMetadata, areLinked AreLinked) probe.Probe {
```

Comment:
Make `getMetadata` optional:

```
func NewABLinker(g *graph.Graph, aManager, aType, bManager, bType string, areLinked AreLinked, getMetadata ...GetMetadata) probe.Probe {
```

And default (in case no parameter was passed) to generating default metadata:

```
Manager: <manager>
RelationType: <aType>
```

(note is still sense to keep bType mandatory as in future in may make sense to introduce the following change to default metadata:

```
Manager: <manager>
RelationType: <aType>2<bType>
```

Reply:
DONE